worker/src/RTC/Codecs/VP8.cpp

```diff
@@ -339,6 +339,12 @@ namespace RTC
 			if (context->GetCurrentTemporalLayer() > context->GetTargetTemporalLayer())
 				context->SetCurrentTemporalLayer(context->GetTargetTemporalLayer());
 
+			// Do not send tlIndex higher than current one.
+			if (this->payloadDescriptor->tlIndex > context->GetCurrentTemporalLayer())
```

Comment:
Don't we have the same check here https://github.com/versatica/mediasoup/blob/c1db7684e6efb323f5ccc67f5fc41b071438c7dc/worker/src/RTC/Codecs/VP8.cpp#L267 ?

Reply:
> Don't we have the same check here https://github.com/versatica/mediasoup/blob/c1db7684e6efb323f5ccc67f5fc41b071438c7dc/worker/src/RTC/Codecs/VP8.cpp#L267 ?

The check you point does not apply to older pictureIds than the last forwarded one (check the block condition). Those older packets are now blindly forwarded.